src/In.ProjectEKA.HipService/Common/LogEvents.cs

```diff
@@ -0,0 +1,7 @@
+namespace In.ProjectEKA.HipService.Common
+{
+    public class LogEvents
```

Comment:
Codacy found an issue: [Add a 'protected' constructor or the 'static' keyword to the class declaration.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=5909857)